simuq/providers.py

```diff
@@ -55,17 +55,15 @@ def compile(self, qs, provider, device, aais, tol=0.01, trotter_num=6, state_pre
         if self.provider == "quera":
             nsite = qs.num_sites
 
-            if aais == "rydberg1d_global":
-                from simuq.aais.rydberg1d_global import GenMach
+            if aais == 'rydberg1d_global' :
+                from simuq.aais.rydberg1d_global import Rydberg1DGlobalQMachineFactory
```

Comment:
Try to keep imports at the top of the file

Reply:
I'll keep this here for now. If the AAIS depends on external libraries then this may not work. I'm planning to do move stuff around when we create more modules